doc/oil-help.md

```diff
@@ -284,7 +284,7 @@ The shell-like syntax accepts the same syntax that a command can:
 
     +  -  *  /   //   %   **
 
-`div` is for integer math, while `/` is for floating point math.
+`div` is for integer math, while `/` is for floating-point math.
```

Comment:
Please remove this whole sentence, as we no longer have `div`.

(I also prefer the unhyphenated version but it's moot :) )